src/pages/Mypage/Mypage.js

```diff
@@ -1,7 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
+import MypageTop from './components/MypageTop';
+import MenuBar from './components/MenuBar';
+import OrderDelivery from './components/OrderDelivery';
+import Subscribes from './components/Subscribes';
+import Likes from './components/Likes';
 import './Mypage.scss';
 
 const Mypage = () => {
-  return <div className="mypage">마이페이지</div>;
+  const [isOpen, setIsOpen] = useState(1);
+
```

Comment:
```suggestion
  const infoList = {
    1 : <OrderDelivery />
    2 : <Subscribes />
    3 : <Likes />
  }
```

이렇게 컴포넌트를 관리하는 객체를 만들어 놓고
menuBar를 눌러서 변경되는 state의 값에 따라 매핑해서 적용할 수 있겠습니다!